src/common/utils.cc

```diff
@@ -119,3 +120,57 @@ void removeDirectory( string const & directory )
 }
 
 } // namespace Utils::Fs
+
+namespace Utils::WebSite {
```

Comment:
Maybe this should stay in `dict/website.cc` as a function because it won't be used else where.

Reply:
I'm going to work on a different PR(open website dictionary in a different tab) ,which may need this function.